indexer-app/src/cmr/indexer/data/index_set.clj

```diff
@@ -416,6 +416,9 @@
   (merge
     {:concept-id (m/stored m/string-field-mapping)
 
+     :native-id m/string-field-mapping
+     :native-id.lowercase m/string-field-mapping
```

Comment:
I remember @chris-durbin telling me that we should be using `m/doc-field`s here, but I don't remember why. I'm not sure if it applies here, but it's worth bringing up for granules, I think